{{ cookiecutter.repo_name }}/{{ cookiecutter.pkg_name }}/config/settings/common.py

```diff
@@ -142,7 +142,7 @@ class Common(Configuration):
     # Don't put anything in this directory yourself; store your static files
     # in apps' "static/" subdirectories and in STATICFILES_DIRS.
     # Example: "/var/www/example.com/static/"
-    STATIC_ROOT = os.path.join(BaseDir.BASE_DIR, 'static_root')
+    STATIC_ROOT = values.Value(os.path.join(BaseDir.BASE_DIR, 'static_root'))
```

Comment:
Please change to [PathValue](https://django-configurations.readthedocs.org/en/stable/values/#configurations.values.PathValue)